Cargo.toml

```diff
@@ -45,3 +46,8 @@ utils = { path = "crates/utils" }
 arc-swap = { version = "1.7.1" }
 num-traits = "0.2"
 lazy_static = "1.4.0"
+aptos-sdk = { git = "https://github.com/Tranduy1dol/aptos-core", branch="zeroize_v1.7" }
```

Comment:
I will create an aptos core fork in sota-zk-labs later, add a todo to replace in the future

Reply:
I did that. Can you review it? Here is the [link](https://github.com/sota-zk-labs/aptos-core)